src/check/arbitrary/ObjectArbitrary.ts

```diff
@@ -533,37 +458,12 @@ function json(constraints?: number | JsonSharedConstraints): Arbitrary<string> {
  *
  * Keys and string values rely on {@link unicode}
  *
- * @remarks Since 0.0.7
- * @public
- */
-function unicodeJson(): Arbitrary<string>;
-/**
- * For any JSON strings with unicode support and a maximal depth
- *
- * Keys and string values rely on {@link unicode}
- *
- * @param maxDepth - Maximal depth of the generated objects
- *
- * @deprecated
- * Superceded by `fc.unicodeJson({maxDepth})` - see {@link https://github.com/dubzzz/fast-check/issues/992 | #992}.
- * Ease the migration with {@link https://github.com/dubzzz/fast-check/tree/main/codemods/unify-signatures | our codemod script}.
+ * @param constraints - Constraints to be applied onto the generated instance (since 2.5.0)
  *
  * @remarks Since 0.0.7
  * @public
  */
-function unicodeJson(maxDepth: number): Arbitrary<string>;
-/**
- * For any JSON strings with unicode support
- *
- * Keys and string values rely on {@link unicode}
- *
- * @param constraints - Constraints to be applied onto the generated instance
- *
- * @remarks Since 2.5.0
- * @public
- */
-function unicodeJson(constraints: JsonSharedConstraints): Arbitrary<unknown>;
-function unicodeJson(constraints?: number | JsonSharedConstraints): Arbitrary<string> {
+function unicodeJson(constraints: JsonSharedConstraints = {}): Arbitrary<unknown> {
```

Comment:
```suggestion
function unicodeJson(constraints: JsonSharedConstraints = {}): Arbitrary<string> {
```

By the way, there is a typo there